### DISCUSSION TITLE
How to Import Datasets  from API datasources?

### DISCUSSION THREAD

**Discussion by rtcpng548** [planetscale/discussion#509] [Q&A]
Just created planetscale dB. I want to get started by importing dataset from API datsource such as Power BI. I want use planetscale as repository for daily dataset imports. I've seen documentation for import from mysql datasource but nothing on API json import. How can I achieve this?

**Comment by orware** [comment]
Hello @rtcpng548,

The easiest approach for what you've described would likely involve a custom script that would act as the intermediate spot to pull from the API you may need data from and perform `INSERT` / `UPDATE` queries against your PlanetScale database.

That custom script could be programmed in just about any language so long as you could communicate with the API and the PlanetScale database. Then you'd likely want to schedule it to run semi-regularly to ensure your data is kept up-to-date.
